server/data/accreditedProgrammesApi/courseClient.test.ts

```diff
@@ -194,7 +194,10 @@ pactWith({ consumer: 'Accredited Programmes UI', provider: 'Accredited Programme
 
   describe('findOfferings', () => {
     const courseOfferings = [
-      courseOfferingFactory.build({ id: '790a2dfe-7de5-4504-bb9c-83e6e53a6537' }),
+      courseOfferingFactory.build({
+        id: '790a2dfe-7de5-4504-bb9c-83e6e53a6537',
+        secondaryContactEmail: 'nobody2-bwn@digital.justice.gov.uk',
+      }),
```

Comment:
Do they both have secondary contact emails on the API now?

Reply:
Haven't updated the API yet, as I'm running locally against the dev branch